src/components/MainNav/DocsSubmenu/index.js

```diff
@@ -11,9 +11,9 @@ export default function Docs({ menu, parentURL }) {
                     <Link className="text-primary hover:text-primary" to={parentURL}>
                         <h1 className="hidden lg:inline-block text-4xl m-0 font-bold">{menu.title}</h1>
                     </Link>
-                    <p className={submenu.section.description('hidden lg:block')}>
+                    <div className={submenu.section.description('hidden lg:block')}>
```

Comment:
To fix this error,

<img width="568" alt="" src="https://user-images.githubusercontent.com/5864173/150375862-986126c4-2856-47f9-a1f6-fe40a706f0d3.png">